packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp

```diff
@@ -840,15 +840,16 @@ uint64_t ec_pairing_variable_gas_cost(const MachineState& m) {
         return gas_cost;
     }
     try {
-        const Tuple* val = &assumeTuple(m.stack[0]);
+        auto val = m.stack[0];
```

Comment:
I think we want `const auto&` to avoid the copy or `const value*` if we need it to be reassignable